haas/client/errors.py

```diff
@@ -0,0 +1,22 @@
+class AuthenticationError(Exception):
+    pass
+
+
+class NotFoundError(Exception):
+    pass
+
+
+class DuplicateError(Exception):
+    pass
+
+
+class BlockedError(Exception):
+    pass
+
+
+class ProjectMismatchError(Exception):
```

Comment:
I wanted to keep the client library confined to its folder. I am not importing from the `haas` folder anything. Is it too bad to let this error.py stay with just one exception defined in it ?